modules/tournament/src/main/TournamentApi.scala

```diff
@@ -201,15 +201,17 @@ final class TournamentApi(
     else if (tour.isCreated) destroy(tour)
     else funit
 
+  private def tourUrl(tourId: String): String = s"https://lichess.org/tournament/$tourId"
```

Comment:
This was copied verbatum from [`PairingSystem.scala`](https://github.com/ornicar/lila/blob/master/modules/tournament/src/main/arena/PairingSystem.scala#L102), should they both be changed to
`: String = s"//lichess.org/tournament/$tourId"`

Reply:
idk, but they should be deduped and moved to `object Tournament`